indexer/src/main/java/au/org/aodn/esindexer/utils/MapperUtils.java

```diff
@@ -59,15 +59,20 @@ public static String mapContactsPosition(CIIndividualPropertyType2 individual) {
      */
     public static ContactsAddressModel mapContactsAddress(CIAddressPropertyType2 address) {
         ContactsAddressModel addressItem = ContactsAddressModel.builder().build();
-        List<String> deliveryPoints = new ArrayList<>();
 
-        address.getCIAddress().getDeliveryPoint().forEach(deliveryPoint -> {
-            String deliveryPointString = deliveryPoint.getCharacterString().getValue().toString();
-            deliveryPoints.add(deliveryPointString != null ? deliveryPointString : "");
-        });
+        List<au.org.aodn.metadata.iso19115_3_2018.CharacterStringPropertyType> v = safeGet(() -> address.getCIAddress().getDeliveryPoint());
```

Comment:
what does `v` stand for?

Reply:
can you rename it with something else easier to read?

Reply:
changed